x-pack/solutions/security/packages/navigation/src/constants.ts

```diff
@@ -53,7 +54,7 @@ export enum ExternalPageName {
   mlNodesOverview = 'ml:nodesOverview',
   mlNodes = 'ml:nodes',
   mlFileUpload = 'ml:fileUpload',
-  mlIndexDataVisualizer = 'ml:indexDataVisualizer',
+  mlIndexDataVisualizer = 'ml:indexDataVisualizer', // TODO: confirm if this is needed for deep search links before removing
```

Comment:
@alvarezmelissa87 This entry is only used to navigate from within the Security Solution pages. If we are not using it, we can remove it.
This is not used in the global search if that's the question. You can test that, remove this line and search for the link :)

Reply:
I see them being used in`x-pack/solutions/security/plugins/security_solution/public/app/solution_navigation/links/sections/ml_links.ts` so I'll delete the comment and keep these constants.